### PR TITLE
ROC-6396: Hidden date picker errors

### PR DESCRIPTION
**Pull request by KiranVarma02**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-6396

### Change description
Hidden date picker errors

### CODE REVIEW

File: src/main/public/js/date-picker.js [hunk 1]
```diff
@@ -86,9 +86,12 @@ const datePicker = {
     const csrf = $("input[name=\"_csrf\"]").val();
 
     let dates = event.dates.map(function(eventDate) { return datePickerUtils.formatDateForData(eventDate)});
+
+    let hasUnavailableDates = $("input[name=hasUnavailableDates]:checked").val()
```
Comment (andrewwa-kainos) [inline src/main/public/js/date-picker.js]: add semi-colon

File: src/main/public/js/date-picker.js [hunk 2]
```diff
@@ -86,9 +86,12 @@ const datePicker = {
...
+    if (hasUnavailableDates === "true") {
     $.post("/case/" + uuid + "/directions-questionnaire/hearing-dates/date-picker/replace", {
```
Comment (andrewwa-kainos) [inline src/main/public/js/date-picker.js]: Indent the body of this conditional

File: src/main/public/js/date-picker.js (hunk 1 above)
Comment (KiranVarma02) [reply src/main/public/js/date-picker.js]: done

File: src/main/public/js/date-picker.js (hunk 2 above)
Comment (KiranVarma02) [reply src/main/public/js/date-picker.js]: done

**Pull request merged**